lib/screens/dashboard.dart

```diff
@@ -1,31 +1,75 @@
+import '../models/Request_Details.dart';
 import 'package:flutter/material.dart';
-import 'addroom.dart';
+import './addroom.dart';
+import './trips_list.dart';
+
 
 class Dashboard extends StatefulWidget {
   @override
   _DashboardState createState() => _DashboardState();
 }
 
 class _DashboardState extends State<Dashboard> {
+
+  final List<RequestDetails> _allRequests = [];
+
+  void _addNewRequest(
+      String rqDest,
+      String rqFinalDest,
+      DateTime startDate,
+      TimeOfDay startTime,
+      DateTime endDate,
+      TimeOfDay endTime,
+      bool privacy) {
+    final newRq = RequestDetails(
+        name: 'Name',
+        id: DateTime.now().toString(),
+        destination: rqDest,
+        finalDestination: rqFinalDest,
+        startDate: startDate,
+        startTime: startTime,
+        endDate: endDate,
+        endTime: endTime,
+        privacy: privacy
+    );
+    setState(() {
+      _allRequests.add(newRq);
+    });
+  }
+  void _startCreatingRequests(BuildContext ctx){
+    showModalBottomSheet(
+      shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.vertical(top: Radius.circular(25.0))
+      ),
+      context: ctx,
+      builder: (_) {
+
+        return GestureDetector(
+          onTap: () {},
+          child: NewRequest(_addNewRequest),
+          behavior: HitTestBehavior.opaque,
+        );
+      },
+    );
+  }
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: Center(
-        child: Text(
-          'Dashboard will be shown here',
-          style: TextStyle(fontSize: 25.0),
-        ),
+      body: Column(
+        children: <Widget>[
+          Container(
+            margin: EdgeInsets.all(5),
+            height: (MediaQuery.of(context).size.height - MediaQuery.of(context).padding.top)*0.84 ,
+            width: double.infinity,
+            child: TripsList(_allRequests),
+          ),
+        ],
```

Comment:
Dashboard screen is showing 
```
A RenderFlex overflowed by 16 pixels on the bottom.
I/flutter (19991): The relevant error-causing widget was:
I/flutter (19991):   Column
I/flutter (19991):   file:///Users/vishal/Desktop/Coding%20B/Flutter/workspace/shareacab/lib/screens/dashboard.dart:59:13
```
Run in the emulator to check this

Reply:
@Ishaan21 Please look into this issue and fix it